animareflection/ui
# Typography

## Description

##### Task link: https://trello.com/c/QLGYe9pL/136-custom-fonts

- Added primary font (Ageo)
- Added code font (Noto Sans Mono)
- Automatically injected fonts via main stylesheet (shipped with library bundle)
- Added introduction MDX story

We can change the fonts to whatever we'd like, I just wanted to get at least 2 in for a thorough reference.

## Test Steps

- Verify custom fonts are automatically applied with no special application code additions (e.g. run `examples/next` project as is and check fonts)
- Verify font can be overridden (e.g. test `globalCss: { "*": { fontFamily: "Times New Roman" } }` in `examples/next/panda.config.ts`)
- Verify Next.js [font optimization](https://nextjs.org/docs/pages/building-your-application/optimizing/fonts) is enabled for Next.js apps
  - This can be tested by running the `examples/next` project (particularly as a build for highest production parity), opening the Network tab in the browser developer tools, and verifying that static CSS assets are delivered locally from `_next/static/...` (e.g. `http://localhost:3000/_next/static/media/ageo-bold-WMPALNCL.b31e05b4.woff2`)
- Verify storybook UI (e.g. sidebar) uses custom primary font
